joomla/joomla-cms
# Just update the timestamps for core sample data

Pull Request for Issue #15961 

### Summary of Changes

This now limit the timestamp updates just to the core sample data.

### Testing Instructions

- download: https://github.com/zero-24/joomla-cms/archive/just_change_core_sampledata.zip
- install as normal joomla with core sample data
- the dates should be updated
- add a sql update file (to be provided by @hendrikbehncke) to the following folder: https://github.com/zero-24/joomla-cms/tree/just_change_core_sampledata/installation/sql/mysql
- confirm the dates are not touched and the same as the values as provided.

### Expected result

The core files are uptodate and the 3rd Party files are not.

### Actual result

The core and 3rd party files are uptodate.

### Documentation Changes Required

It should be noted somewhere that this is only happening now for the core.
